crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -295,14 +295,17 @@ impl OlmMachine {
         self.store.device_display_name().await
     }
 
-    /// Get all the tracked users we know about
+    /// Get the list of "tracked users".
+    ///
+    /// See [`update_tracked_users`](#method.update_tracked_users) for more
+    /// information.
     pub fn tracked_users(&self) -> HashSet<OwnedUserId> {
         self.store.tracked_users()
     }
 
     /// Get the outgoing requests that need to be sent out.
     ///
-    /// This returns a list of `OutGoingRequest`, those requests need to be sent
+    /// This returns a list of `OutgoingRequest`. Those requests need to be sent
```

Comment:
Let's link to the type as well: 

```suggestion
    /// This returns a list of [`OutgoingRequest`]. Those requests need to be sent
```